### ISSUE TITLE
Wrong `nft add element` command is generated when modifying rules

### ISSUE THREAD

**Issue by jay7x** [CloudFire-LLC/cloudfire#1867]
**Describe the bug**
Firezone emits `nft` command to modify a set when one addings/remove rules via WebUI or REST API.
This command has wrong `port_range` parameter. See the logs below.

**To Reproduce**
Install Firezone v0.7.32 (not sure about older versions) and create a rule via WebUI or REST API.

**Expected behavior**

**Screenshots / Logs**

```
17:54:38.530 erl_level=error domain=otp file=gen_server.erl function=error_info/8 line=1402 mfa=:gen_server.error_info/8 module=gen_server pid=<0.490.0> registered_name=fz_wall_server [error] GenServer :fz_wall_server terminating
** (RuntimeError)   Error executing command   nft 'add element inet firezone ip_accept_layer4 { 1.2.3.4 . tcp . [23000,24001) }'
.
  Exit code: 1
  Error message:
  Error: syntax error, unexpected '['
add element inet firezone ip_accept_layer4 { 1.2.3.4 . tcp . [23000,24001) }
                                                             ^


    (fz_wall 0.7.32) lib/fz_wall/shell.ex:24: FzWall.Shell.exec/2
    (fz_wall 0.7.32) lib/fz_wall/server.ex:76: FzWall.Server.add_rule/2
    (fz_wall 0.7.32) lib/fz_wall/server.ex:24: FzWall.Server.handle_call/3
    (stdlib 4.2) gen_server.erl:1149: :gen_server.try_handle_call/4
    (stdlib 4.2) gen_server.erl:1178: :gen_server.handle_msg/6
    (stdlib 4.2) proc_lib.erl:240: :proc_lib.init_p_do_apply/3
Last message (from #PID<0.355.0>): {:add_rule, %{action: "accept", destination: "1.2.3.4", port_range: "[23000,24001)", port_type: "tcp", user_id: nil}}
17:54:38.531 erl_level=error domain=otp file=gen_server.erl function=error_info/8 line=1402 mfa=:gen_server.error_info/8 module=gen_server pid=<0.355.0> [error] GenServer #PID<0.355.0> terminating
** (stop) exited in: GenServer.call(#PID<0.490.0>, {:add_rule, %{action: "accept", destination: "1.2.3.4", port_range: "[23000,24001)", port_type: "tcp", user_id: nil}}, 5000)
    ** (EXIT) an exception was raised:
        ** (RuntimeError)   Error executing command   nft 'add element inet firezone ip_accept_layer4 { 1.2.3.4 . tcp . [23000,24001) }'
.
  Exit code: 1
  Error message:
  Error: syntax error, unexpected '['
add element inet firezone ip_accept_layer4 { 1.2.3.4 . tcp . [23000,24001) }
                                                             ^


            (fz_wall 0.7.32) lib/fz_wall/shell.ex:24: FzWall.Shell.exec/2
            (fz_wall 0.7.32) lib/fz_wall/server.ex:76: FzWall.Server.add_rule/2
            (fz_wall 0.7.32) lib/fz_wall/server.ex:24: FzWall.Server.handle_call/3
            (stdlib 4.2) gen_server.erl:1149: :gen_server.try_handle_call/4
            (stdlib 4.2) gen_server.erl:1178: :gen_server.handle_msg/6
            (stdlib 4.2) proc_lib.erl:240: :proc_lib.init_p_do_apply/3
    (elixir 1.14.3) lib/gen_server.ex:1038: GenServer.call/3
    (fz_http 0.7.32) lib/fz_http/repo/notifier.ex:25: FzHttp.Repo.Notifier.handle_info/2
    (stdlib 4.2) gen_server.erl:1123: :gen_server.try_dispatch/4
    (stdlib 4.2) gen_server.erl:1200: :gen_server.handle_msg/6
    (stdlib 4.2) proc_lib.erl:240: :proc_lib.init_p_do_apply/3
Last message: {:notification, #PID<0.354.0>, #Reference<0.1827560603.2326003713.62666>, "rules_changed", "{\"op\" : \"INSERT\", \"row\" : {\"destination\":\"1.2.3.4\",\"action\":\"accept\",\"inserted_at\":\"2023-08-07T17:54:38.50522\",\"updated_at\":\"2023-08-07T17:54:38.50522\",\"id\":\"adc8014a-2eb0-4099-b03c-88f8a4410220\",\"port_range\":\"[23000,24001)\",\"port_type\":\"tcp\",\"user_id\":null}}"}
```

**Platform (please complete the following information):**
 - Firezone Version: 0.7.32
 - OS: Linux
 - Distro: Ubuntu 22.04
 - Kernel version: 5.19.0-1030-gcp
 - Deployment method: Docker

**Additional context**

Moreover, on 4th rule added (or deleted) via WebUI or REST API firezone process terminates. Somehow I was unable to reproduce this with `curl` but it's pretty reproducible via WebUI when deleting 4 rules in a row quickly. Also I can reproduce it with my Ansible playbook which creates missing rules.

I'm not sure what is the reason but logs are the same as above with the following messages in the end:

```
18:27:33.046 erl_level=info application=phoenix domain=elixir file=lib/phoenix/socket/pool_supervisor.ex function=terminate/2 line=114 mfa=Phoenix.Socket.PoolDrainer.terminate/2 module=Phoenix.Socket.PoolDrainer pid=<0.477.0> [info] Shutting down 1 sockets in 1 rounds of 2000ms
18:27:33.057 erl_level=info application=phoenix domain=elixir file=lib/phoenix/socket/pool_supervisor.ex function=terminate/2 line=114 mfa=Phoenix.Socket.PoolDrainer.terminate/2 module=Phoenix.Socket.PoolDrainer pid=<0.476.0> [info] Shutting down 2 sockets in 1 rounds of 2000ms
{"Kernel pid terminated",application_controller,"{application_terminated,fz_wall,shutdown}"}
Kernel pid terminated (application_controller) ({application_terminated,fz_wall,shutdown})
```

**Comment by jay7x** [comment]
Ah, forgot to mention that I'm doing this either by 1) querying phoenix on port 13000 via http -or- 2) Nginx which terminates TLS and go to phoenix. No caddy involved.

**Comment by jamilbk** [comment]
Should be fixed in 0.7.35. Please reopen if you're still experiencing this issue!
